packages/blockchain-wallet-v4-frontend/package.json

```diff
@@ -74,11 +74,12 @@
   },
   "dependencies": {
     "@blockchain-com/components": "6.1.1",
-    "@ledgerhq/hw-app-eth": "4.32.0",
     "@ledgerhq/hw-app-btc": "4.33.7",
+    "@ledgerhq/hw-app-eth": "4.32.0",
     "@ledgerhq/hw-app-str": "4.32.0",
     "@ledgerhq/hw-transport-u2f": "4.32.0",
     "@veriff/incontext-sdk": "0.3.0",
+    "babel-plugin-styled-components": "^1.11.1",
```

Comment:
Can you pin the dependency version please? i.e. remove `^`

Reply:
@schnogz done ;)